packages/sources/snowflake/src/index.ts

```diff
@@ -0,0 +1,6 @@
+import { expose } from '@chainlink/ea-bootstrap'
+import { makeExecute, endpointSelector } from './adapter'
+import { makeConfig, NAME } from './config'
+
+const { server } = expose(NAME, makeExecute(), undefined, endpointSelector)
+export { NAME, makeExecute, makeConfig, server }
```

Comment:
This will be useful once you add the ResponseSchema interface

```suggestion
import { makeConfig, NAME } from './config'
import * as types from './endpoint'

const { server } = expose(NAME, makeExecute(), undefined, endpointSelector)
export { NAME, makeExecute, makeConfig, server, types }
```